src/components/Packages/PackagePanel.js

```diff
@@ -89,15 +89,15 @@ class PackagePanel extends Component {
 								{/* eslint-disable-next-line */} 
 								<a className="d-block pb-1" onClick={this.handleMetadataClick}>Show package metadata</a>
 							</Col>
-							{this.props.uploadPackage.downloadable &&
+							{this.props.uploadPackage.state.state === "UPLOAD_SUCCEEDED" &&
```

Comment:
I expect this bit will move into the PackagePanelStateText when we have implemented the states around DVC QC and begin showing those in the interface rather than just a download button.